bucket/iddsampledriver-ge9-np.json

```diff
@@ -0,0 +1,52 @@
+{
+    "version": "0.0.1.2",
+    "description": "Windows Indirect Display Driver using the IddCx class extension driver. Virtual monitors for Windows 10/11.",
+    "homepage": "https://github.com/ge9/IddSampleDriver",
+    "license": "Unknown",
+    "notes": "Use \"C:\\IddSampleDriver\\option.txt\" to set the number of monitors and resolutions, then restart the driver.",
```

Comment:
How do I "restart the driver"? Do you mean re-loading it in *Device Manager*?

Reply:
Yes, or disabling and enabling it via other methods.

Reply:
I suggest adding more info like `Then, restart the driver via Device Manager or (some app)`

Reply:
The Device Manager is a general way to do it, it's up to them now if they use some other app 👍